--- jquery-2.2.4.js	2019-04-01 08:39:29.000000000 +0200
+++ jquery-2.2.4.js	2019-04-01 09:02:39.000000000 +0200
@@ -208,8 +208,9 @@ jQuery.extend = jQuery.fn.extend = function() {
 			for ( name in options ) {
				src = target[ name ];
 				copy = options[ name ];
 
+				// Prevent Object.prototype pollution
 				// Prevent never-ending loop
-				if ( target === copy ) {
+				if ( name === "__proto__" || target === copy ) {
 					continue;
 				}
 
